testdata/dnn/.gitignore

```diff
@@ -1,4 +1,3 @@
-torch/*.txt
```

Comment:
Looks like this is needed for https://github.com/opencv/opencv_extra/blob/master/testdata/dnn/generate_torch_models.py script.